book/2-begin/server/models/User.js

```diff
@@ -1,41 +0,0 @@
-const mongoose = require('mongoose');
```

Comment:
Whoops did we mean to remove the work done for chapter 2?

Reply:
I did not!  Not sure what happened here.  Thank you for pointing this out.